templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails serve Puma -p $PORT -c ./config/puma.rb
```

Comment:
Maybe some details here: http://stackoverflow.com/a/21218048 That answer
makes sense to me but I wonder why Heroku's docs are different, if there's
anything platform-specific that makes the Heroku docs version better on
Heroku. Maybe @schneems or @hone knows?

On Mon, Jan 26, 2015 at 3:42 PM, Caleb Thompson <notifications@github.com>
wrote:

> In templates/Procfile
> <https://github.com/thoughtbot/suspenders/pull/501#discussion_r23575253>:
>
> > @@ -1,2 +1,2 @@
> > -web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
> > +web: bundle exec rails serve Puma -p $PORT -c ./config/puma.rb
>
> I pulled it from https://github.com/puma/puma#rails.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/501/files#r23575253>.
>